eth_interface.md

```diff
@@ -31,12 +31,18 @@ Subtracts an amount to the gas counter
 
 ## getAddress
 
-Gets address of currently executing account and loads it into memory at
-the given offset.
+```wast
```

Comment:
Why is this wast example needed?

Reply:
I though that providing a wast function declaration would look nice.

Reply:
is `i32ptr` a valid alias of `i32` in wast?

Reply:
No. It's only the alias here for easier reading.